### PR TITLE
added linea-sdk documentation

### PR DESCRIPTION
**Pull request by cedrabner**
i expect some more info from engineering to come back but i think we have a good base. Creating PR for visibility.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,6 +29,7 @@
     "**/*.{md,mdx,ts,js,tsx,jsx,json}": "npm run format"
   },
   "dependencies": {
+    "@consensys/linea-sdk": "^0.1.5",
```
Comment (emilyJLin95) [inline package.json:32]: Is this needed?

Comment (cedrabner) [reply package.json:32]: nope

Comment (emilyJLin95) [reply package.json]: If so, should we delete this change?

**Review by mapachurro — APPROVED** [review]
Super psyched to see this launch.

**Pull request merged**